### ISSUE TITLE
SFDX: Authorize an org doesn't work and I didn't get any clear error notifaction.

### ISSUE THREAD

**Issue by SoOrAj247** [forcedotcom/salesforcedx-vscode#3824]
SFDX: Authorize an org command's not working

**Comment by randi274** [comment]
Hi @SoOrAj247 are you also having issues with the push and pull commands, like #3816?

**Comment by julianperezpesce** [comment]
I'm having the same problem on Linux. I can deploy a retrieve, but I can NOT "update checkpoint on a Org" for example. Some things work, another it doesn't. Already try install/uninstall, update sfdx, created new playgrounds, projects, etc nothing work. My problem appear after a trailhead were I had to create an Scratch Org. I'm on a bootcamp, and now I'm stuck. Don't know what to do, please help me!!!

**Comment by gbockus-sf** [comment]
Hi @julianperezpesce 
A few questions: 
Are you able to run sfdx from the command line? `sfdx force:org:open -u <yourUserName>`
If so can you try opening vscode from that command line and see if the command behave as expected? `code .` from your sf project directory.
Thanks

**Comment by julianperezpesce** [comment]
> Hi @julianperezpesce A few questions: Are you able to run sfdx from the command line? `sfdx force:org:open -u <yourUserName>` If so can you try opening vscode from that command line and see if the commands behave as expected? `code .` from your sf project directory. Thanks

Hi, I already fixed. 
1- sfdx force:org:open -u <yourUserName> WORKED
2- code . WORKED

But that didn't fix anything. I'm using Linux, the problem was, for some reason a change in the Read, Write permission of .sfdx json files. That was producing corrupteds json files. I made a new SO install. Another thing I discover, I was using JDK 8, and VSC always give an error with Apex debugger a Yellow notification. Now I'm using JDK11(temurin) and everything works perfectly.
Thank you for your help!!!

PS: Sorry for my bad english.
